test/libsolidity/syntaxTests/functionTypes/internal_function_array_as_external_parameter_in_library_external.sol

```diff
@@ -0,0 +1,7 @@
+library L {
+    // Used to cause internal error
+    function f(function(uint) internal returns (uint)[] memory x) public {
```

Comment:
Do we have a test that uses `storage` here?

Reply:
I'll add one